docs/tutorials/tutorial2.md

```diff
@@ -165,6 +165,18 @@ value.
 
     `_addressLine1 = "address1"`
 
+When renaming the name of a field referring to a primary key (for example
```

Comment:
I'd say "referring to a foreign key" (even though you use `PrimaryKey` for this) as primary key suggests the key of the table in question.